libs/minecraft/PosUtils.arucas

```diff
@@ -1,19 +1,34 @@
+import * from Minecraft;
+import ImmutableList from util.Collections;
+enum Axis {
+    static var AXIS_X = ImmutableList.of(Direction.EAST, Direction.WEST);
+    static var AXIS_Y = ImmutableList.of(Direction.UP, Direction.DOWN);
+    static var AXIS_Z = ImmutableList.of(Direction.SOUTH, Direction.NORTH);
+    static var NOTHING = ImmutableList.of(Direction.NONE);
```

Comment:
This shouldn't compile. Have you tried this?
Enum constants should always be **first**.

Reply:
now it compiles but there're... more problems